cwl_logs/serverless.yml

```diff
@@ -0,0 +1,66 @@
+service: DE-Cloudwatch-Logs-to-Elasticsearch
+
+plugins:
+  - serverless-pseudo-parameters
+
+custom:
+  region: ${self:provider.region}
+  stage: ${opt:stage, self:provider.stage}
+  esClusterArn: ${env:ES_CLUSTER_ARN}
+  esHost: ${env:ES_HOST}
+  esPort: ${env:ES_PORT}
+  monitoredLogGroup: ${env:MONITORED_LOG_GROUP}
+
+package:
+  individually: true
+  exclude:
+    - ".*/**"
+    - "**/**"
+    - "*"
+
+provider:
+  name: aws
+  runtime: python3.8
+  timeout: 60
+  region: us-east-1
+  stage: dev
+  versionFunctions: true
+  deploymentBucket:
+    name: ${env:SLS_DEPLOYMENT_BUCKET}  
+
+  iamRoleStatements:
```

Comment:
This seems over-permissioned for it's job. Does it really need more than read access on CWL, and Put access on ES?